### ISSUE TITLE
SC1125 "Invalid key=value pair?" missing wiki page

### ISSUE THREAD

**Issue by Edward-Knight** [koalaman/shellcheck#1979]
#### For bugs
- [x] Rule Id (if any, e.g. SC1000): SC1125
- [x] My shellcheck version (`shellcheck --version` or "online"): online
- [x] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

This code:
```sh
#!/usr/bin/env sh

variable=${VARIABLE}

set -o nounset

echo "${variable}"
```
produced the error:
```
SC2153: Possible misspelling: VARIABLE may not be assigned, but variable is.
```

I added the disable directive, with a note:
```sh
#!/usr/bin/env sh

# shellcheck disable=SC2153 (variable not a misspelling)
variable=${VARIABLE}

set -o nounset

echo "${variable}"
```
But this produced a further error:
```
SC1125: Invalid key=value pair? Ignoring the rest of this directive starting here.
```
Which doesn't have a wiki page. On reading the rest of the wiki, this page had the useful information about how to comment a directive:
https://github.com/koalaman/shellcheck/wiki/Directive#documenting-directive-use
> The comment can be added on the preceding line. This is the recommended option for long comments.
> The comment can also be added at the end of the directive line. This is the recommended option for short comments.

#### Here's what shellcheck currently says:

No wiki page (option to create new page for SC1125).

#### Here's what I wanted or expected to see:

A wiki page explaining directive formatting / commenting style.

**Comment by brother** [comment]
Done. https://github.com/koalaman/shellcheck/wiki/SC1125

**Comment by Edward-Knight** [comment]
Thanks, looks good 👍

**Comment by yigtuyumz** [comment]
I am trying to enable a ShellCheck directive, which i disabled before in the same script file.
`
#!/bin/sh
################################################################################
# shellcheck disable=SC2016,SC2028

some code here......

# shellcheck enable=SC2016,SC2028
^^^^^^^^^^^^^^^^^^^^^^^ causes the SC 1125
################################################################################
`

What is the true syntax for achieve something like this??
